csrankings.csv

```diff
@@ -10513,6 +10515,7 @@ Laura M. Haas,University of Massachusetts Amherst,http://www.umass.edu/newsoffic
 Laura Pozzi,Università della Svizzera italiana,http://www.inf.usi.ch/faculty/pozzi,q59L86wAAAAJ
 Laura Ricci,University of Pisa,http://www.di.unipi.it/~ricci,ODbzAR4AAAAJ
 Laura Ruotsalainen,University of Helsinki,https://researchportal.helsinki.fi/en/persons/laura-ruotsalainen,oKWnTMgAAAAJ
+Laura Sanità,University of Waterloo,http://www.math.uwaterloo.ca/~lsanita/,odfHRTcAAAAJ

```

Comment:
Not CS, not affiliated.
https://cs.uwaterloo.ca/about/people/group/53
https://cs.uwaterloo.ca/about/people/group/49